style.css

```diff
@@ -1,53 +1,138 @@
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #3e3c3c;
+}
+
+.hamburger {
+  display: none;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: white;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-item {
+  margin-left: 5rem;
+}
+
+.nav-links {
+  font-size: 1.25rem;
+  font-weight: 400;
+  color: white;
+  padding-right: 1.0625rem;
+}
+
+.nav-links:hover {
+  color: dodgerblue;
+}
+
+.nav-logo {
+  font-size: 1.875rem;
+  font-weight: 500;
+  color: white;
+  margin-left: 12.5rem;
+}
+
 @media (max-width: 768px) {
-  body {
-    height: fit-content;
-    left: 0;
-    top: 5.75rem;
-    border-radius: 0;
+  .navbar {
+    background: #3e3c3c;
+    width: 30.8125rem;
   }
 
-  .nav-container {
-    display: flex;
-    background-color: #3a3c39;
-    height: 3rem;
-    width: 29.75rem;
-    left: 0;
-    top: 2.75rem;
-    border-radius: 0;
+  .nav-links {
+    padding-right: 18.75rem;
+    font-size: 2rem;
+    color: #fff5e1;
   }
 
-  .shortcut {
-    display: none;
+  .nav-logo {
+    font-size: 1.875rem;
+    font-weight: 500;
+    color: white;
+    margin-left: 1.5rem;
+  }
+
+  hr {
+    color: gray;
+    width: 28.125rem;
+    background-color: gray;
+    margin-right: 1.25rem;
+  }
+
+  .nav-menu {
+    position: fixed;
+    right: -110%;
+    top: 5rem;
+    flex-direction: column;
+    background-color: #262626;
+    width: 100%;
+    text-align: center;
+    transition: 0.3s;
+    box-shadow: 0 10px 27px rgba(0, 0, 0, 0.05);
+    margin-left: 20px;
   }
 
-  .shortcut-mail {
+  .email-icon {
     display: none;
   }
 
-  .burger {
-    height: 0.75rem;
-    width: 1.125rem;
-    left: 0.1875rem;
-    padding-right: 1.8125rem;
-    top: 0.375rem;
-    margin-top: 0.75rem;
-    margin-right: 1.0625rem;
-    margin-bottom: 1.1875rem;
-    border-radius: 0;
+  .nav-menu.active {
+    right: 0;
+    padding-left: 0.75rem;
+    height: 43.75rem;
+    gap: 0.625rem;
+    width: 29.6875rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
   }
```

Comment:
- [ ] You did a good job on the functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible. Please fix the below issues and make sure your design matches the template.
 - Please make sure the mobile menu occupies 100% of the height, Right now the mobile header is visible.
 - Please use the correct background color for mobile menu `#3C3A39;`.
 - Please change the color for the close(X) button and reduce button weight
 - Please increase the font-weight for mobile menu options.
 - Please reduce the weight of horizontal lines between menu options.
 - Please make sure all menu options start at same line(Vertically).

### Your design

![image](https://user-images.githubusercontent.com/83803414/167853140-a125a0c9-78e2-4940-b63e-f97d526d2598.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167853187-d27cbd8b-9f5e-47c0-9631-5df2a3a50340.png)